### PR TITLE
added Nirmal Silwal profile in list

### PR DESCRIPTION
**Pull request by NirmalSilwal**
updated data.json with user details of [NirmalSilwal profile](https://github.com/NirmalSilwal)

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -178,5 +178,11 @@
         "githubUsername": "aniketsingh98571",
         "imageUrl": "https://user-images.githubusercontent.com/63097956/93450123-351c2600-f8f3-11ea-8964-bd44fd12dc0f.png",
         "issueNumber": 88
-    }
+    },
+    {
+    "name": "Nirmal Silwal",
+    "githubUsername": "NirmalSilwal"
```
Comment (eddiejaoude) [inline data.json:184]: Missing comma, should be

```suggestion
    "githubUsername": "NirmalSilwal",
```

File: data.json [hunk 2]
```diff
@@ -178,5 +178,11 @@
...
+    "imageUrl": "https://user-images.githubusercontent.com/31946761/93670031-aefc0d00-fab5-11ea-9ca9-cc4bfe73fa83.png",
+    "issueNumber": 93
+}
```
Comment (eddiejaoude) [inline data.json:187]: Indention is incorrect

File: data.json [hunk 3]
```diff
@@ -178,5 +178,11 @@
         "githubUsername": "aniketsingh98571",
         "imageUrl": "https://user-images.githubusercontent.com/63097956/93450123-351c2600-f8f3-11ea-8964-bd44fd12dc0f.png",
         "issueNumber": 88
-    }
+    },
+    {
+    "name": "Nirmal Silwal",
```
Comment (eddiejaoude) [inline data.json:183]: Indentation is incorrect

**Comment by eddiejaoude** [comment]
Looks good, I made a few small comments

File: data.json (hunk 1 above)
Comment (NirmalSilwal) [reply data.json:184]: My bad! thanks for pointing it out

File: data.json (hunk 2 above)
Comment (NirmalSilwal) [reply data.json:187]: okay, will rearranged it

**Review by eddiejaoude — APPROVED** [review]
Indentation is not quite right, but it can be fixed in another PR

**Comment by NirmalSilwal** [comment]
okay, will do another PR

**Comment by eddiejaoude** [comment]
Ok great 👍 .

**Pull request merged**